packages/itwinui-css/src/input/input-flex-container.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-input-flex-container {
+  --_iui-input-background-color: var(--iui-color-background);
+  --_iui-input-border-color: var(--iui-color-border-foreground);
+  --_iui-input-text-color: var(--iui-color-text);
+
+  @include iui-reset;
+  @include iui-focus($offset: -2px, $thickness: 2px);
+  width: 100%;
+  font-family: inherit;
+  font-size: var(--iui-font-size-1);
+  font-weight: var(--iui-font-weight-normal);
+  line-height: var(--iui-size-l);
+  border-radius: var(--iui-border-radius-1);
+  appearance: none;
+  height: var(--_iui-input-min-height);
+  padding-block: var(--_iui-input-padding-block);
+  padding-inline: var(--iui-size-s);
+  color: var(--_iui-input-text-color);
+  background-color: var(--_iui-input-background-color);
+  border: 1px solid var(--_iui-input-border-color);
+  transition: border-color var(--iui-duration-1) ease-out;
+  @include iui-input-size;
+
+  right: unset;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+
+  > input {
+    padding-left: var(--iui-size-s);
+    width: 100%; // Take all available space
+    color: var(--iui-color-text);
+    font-family: inherit;
+    font-size: var(--iui-font-size-1);
+    font-weight: var(--iui-font-weight-normal);
+    line-height: var(--iui-size-l);
+
+    &::placeholder {
+      --_iui-input-text-color: var(--iui-color-text-disabled);
+      user-select: none;
+      color: var(--_iui-input-text-color);
+    }
+
+    &:autofill {
+      --_iui-input-background-color: var(--iui-color-background-accent-muted);
+      --_iui-input-border-color: var(--iui-color-border-accent);
+      --_iui-input-text-color: var(--iui-color-text-accent);
+    }
+
+    &[disabled] {
+      --_iui-input-background-color: var(--iui-color-background-disabled);
```

Comment:
The disabled searchbox is not getting a darker background color. The background is the same color as the enabled inputs.